clmm/theory/numcosmo.py

```diff
@@ -169,8 +169,8 @@ def _eval_convergence(self, r_proj, z_cl, z_src):
             self.hdpm, self.cosmo.be_cosmo, r_proj, z_src, z_cl, z_cl)
         return np.vectorize(func)(r_proj, z_src, z_cl)
 
-    def _eval_reduced_tangential_shear(self, r_proj, z_cl, z_src):
-        """"eval reduced tangential shear"""
+    def _eval_reduced_tangential_shear_sp(self, r_proj, z_cl, z_src):
+        """"eval reduced tangential shear single plane"""
```

Comment:
Maybe make the docstring a bit more explicit, e.g. "considering a single redshift plane for background sources" ?

Reply:
Done.

Reply:
Done.